pkg/moq/moq_test.go

```diff
@@ -685,3 +685,44 @@ func normalize(d []byte) []byte {
 	d = bytes.Replace(d, []byte{13}, []byte{10}, -1)
 	return d
 }
+
+func TestShadowTypesArgs(t *testing.T) {
```

Comment:
Could you convert this to a test case under `TestMockGolden`?

Reply:
Done. Let me know if it looks OK.